scripts/csv_importer.py

```diff
@@ -0,0 +1,273 @@
+        "                %(final_visibility)s, %(items_default_template_id)s, %(children_display_configuration_id)s,"
+        "                %(items_display_configuration_id)s, %(cached_values)s)"
+    )
+    cursor.execute(insert_statement, collection_data)
+
+    return collection_data["id"]
+
+
+def insert_datum(owner_id, item_id, datum_name, datum_value, datum_position, visibility):
+    datum_data = {
+        "id": generate_id(),
+        "item_id": item_id,
+        "owner_id": owner_id,
+        "type": "text",  # todo?
+        "label": datum_name,
+        "value": datum_value,
+        "position": datum_position,
+        "created_at": get_current_time(),
+        "visibility": visibility
+    }
+    insert_statement = (
+        "INSERT INTO "
+        "koi_datum (id, item_id, owner_id, type, label,"
+        "           value, position, created_at, visibility) "
+        "values    (%(id)s, %(item_id)s, %(owner_id)s, %(type)s, %(label)s,"
+        "           %(value)s, %(position)s, %(created_at)s, %(visibility)s)"
+    )
+    cursor.execute(insert_statement, datum_data)
+
+    return datum_data["id"]
+
+
+def insert_item(owner_id, collection_id, item_name):
+    item_data = {
+        "id": generate_id(),
+        "collection_id": collection_id,
+        "owner_id": owner_id,
+        "name": item_name,
+        "quantity": 1,  # todo?
+        "seen_counter": 0,
+        "visibility": "public",
+        "created_at": get_current_time(),
+        "parent_visibility": "public",  # todo this should not be hardcoded
+        "final_visibility": "public"  # todo this should not be hardcoded
+    }
+    insert_statement = (
+        "INSERT INTO "
+        "koi_item (id, collection_id, owner_id, name, quantity, seen_counter, visibility,"
+        "          created_at, parent_visibility, final_visibility) "
+        "values   (%(id)s, %(collection_id)s, %(owner_id)s, %(name)s, %(quantity)s, %(seen_counter)s, %(visibility)s,"
+        "          %(created_at)s, %(parent_visibility)s, %(final_visibility)s)"
+    )
+    cursor.execute(insert_statement, item_data)
+
+    return item_data["id"]
+
+
+def insert_log(owner_id, object_id, object_name, object_class):
+    log_data = {
+        "id": generate_id(),
+        "owner_id": owner_id,
+        "type": "create",
+        "logged_at": get_current_time(),
+        "object_id": object_id,
+        "object_label": object_name,
+        "object_class": object_class
+    }
+    insert_statement = (
+        "INSERT INTO "
+        "koi_log (id, owner_id, type, logged_at, object_id, object_label, object_class) "
+        "values  (%(id)s, %(owner_id)s, %(type)s, %(logged_at)s, %(object_id)s, %(object_label)s, %(object_class)s)"
+    )
+    cursor.execute(insert_statement, log_data)
+
+    return log_data["id"]
+
+
+def main(file_path, collection):
+    items = load_data(file_path)
+
+    headers = [item for item in items[0].keys() if item.strip() != '']
+    name_field = headers[0]
```

Comment:
Proposed to use `headers[1]` by default

Please consider

Reply:
Changed so the name has to always be explicitly provided by user. Is that ok?

Reply:
Hm, for me it depends.
If in general we use "Nazwa" then nazwa should be default (I think it's [1] in every sheet isn't it?) to simplify usage of the script.

Please consider

Reply:
Ok, didn't look at the worksheet tbh.
Thanks!